notebooks/MAST/TESS/beginner_how_to_use_dvt/beginner_how_to_use_dvt.ipynb

```diff
@@ -0,0 +1,242 @@
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fits.getdata(dvt_file, ext=1).columns"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In addition to the timestamps in TBJD format, there is a column containing the times phased to the orbital period of the signal, and there are several columns of fluxes.  LC_INIT is the \"unwhitened\" fluxes, LC_WHITE are the \"whitened\" fluxes.  The MODEL_INIT and MODEL_WHITE fluxes are the corresponding model fluxes based on the best fit to the signal."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Plotting The Timeseries Fluxes.\n",
+    "\n",
+    "Let's open the FITS file and extract some metadata from the headers, and also store some of the columns from the first TCE signal for use later when we plot the results."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with fits.open(dvt_file, mode=\"readonly\") as hdulist:\n",
+    "    \n",
+    "    # Extract stellar parameters from the primary header.  We'll get the effective temperature, surface gravity,\n",
+    "    # and TESS magnitude.\n",
+    "    star_teff = hdulist[0].header['TEFF']\n",
+    "    star_logg = hdulist[0].header['LOGG']\n",
+    "    star_tmag = hdulist[0].header['TESSMAG']\n",
+    "    \n",
+    "    # Extract some of the fit parameters for the first TCE.  These are stored in the FITS header of the first\n",
+    "    # extension.\n",
+    "    period = hdulist[1].header['TPERIOD']\n",
+    "    duration = hdulist[1].header['TDUR']\n",
+    "    epoch = hdulist[1].header['TEPOCH']\n",
+    "    depth = hdulist[1].header['TDEPTH']\n",
+    "    \n",
+    "    # Extract some of the columns of interest for the first TCE signal.  These are stored in the binary FITS table\n",
+    "    # in the first extension.  We'll extract the timestamps in TBJD, phase, initial fluxes, and corresponding\n",
+    "    # model fluxes.\n",
+    "    times = hdulist[1].data['TIME']\n",
+    "    phases = hdulist[1].data['PHASE']\n",
+    "    fluxes_init = hdulist[1].data['LC_INIT']\n",
+    "    model_fluxes_init = hdulist[1].data['MODEL_INIT']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's make a plot of the detrended fluxes and model fluxes vs. orbital phase."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# First sort the phase and flux arrays by phase so we can draw the connecting lines between points.\n",
+    "sort_indexes = np.argsort(phases)\n",
+    "\n",
+    "# Start figure and axis.\n",
+    "fig, ax = plt.subplots()\n",
+    "\n",
+    "# Plot the detrended fluxes as black circles.  We will plot them in sorted order.\n",
+    "ax.plot(phases[sort_indexes], fluxes_init[sort_indexes], 'ko')\n",
```

Comment:
Suggest adding a smaller markersize, maybe 1 or 2? (Purely cosmetic suggestion, feel free to ignore).